scripts/pipeline/index.html

```diff
@@ -0,0 +1,9 @@
+<html>
```

Comment:
Similar to appspec.yaml. Would be good if all workflow related files move to ".github/workflow" folder.